packages/block-editor/src/dropdown-input/attributes.js

```diff
@@ -24,6 +19,10 @@ export default {
 		type: 'boolean',
 		default: false,
 	},
+	multipleChoice: {
```

Comment:
I wonder if we should stick to the Multiple Choice block attributes. We defined there `minimumChoices` and `maximumChoices` which derive into the **choiceType** of the question. Then again, that would force us to raise the level of complexity on this block's Toolbar/Sidebar.

Reply:
I think this is a 'feature'. When allowing multiple choices, do we want to limit the min/max choices?
If so, indeed, we need to make the sidebar similar to the MC Question.
Otherwise, I'd stick with just a boolean flag and adjust the mapping on the backend, like the Matrix block.
I think @digitalwaveride thoughts will be very appreciated here. 🙂

Reply:
Agree that it's a "feature". When we were working on the MC block, we had this discussion with @ice9js and he made this point: having max/min settings would easily override the need for `multiple` or even `mandatory` (this is something that happens on backend actually):

```
// If mandatory then we always need at least 1 option
if (
	$this->is_mandatory() &&
	$this->min_options == 0
) {
	$this->min_options = 1;
}
...
// If we have a minimum then we also have a mandatory question
if ( $this->min_options > 0 )
	$this->is_mandatory( true );
```
So, as we figured, instead of adding the property "multiple", we added min/max thinking how it would, at some point, make sense for the user to have such option. Reason being, if we added both props/attributes (min/max and multiple), eventually those could collide and a more complex resolving code would be needed. Ex: choose single selection AND minimum choices 2, choose multiple selection and max options 1. 

And, TBH, I think it makes sense and also adds for consistency (as we ARE extending/sharing code from MC block/question).

All this said, we could do this on a revisit, but since we're here...

Design-wise I'm not sure what was the original intention, any opinion/advise @digitalwaveride ?

Reply:
As mentioned in our call today, if we can "steal" the same functionality from the MC block relatively easily, then let's do this. Otherwise, let's postpone that additional feature for now.